pysteps/blending/linear_blending.py

```diff
@@ -22,6 +22,7 @@
 import numpy as np
```

Comment:
Perhaps we should add some explanation about the saliency-based blending method in the docstring above (so the first lines of the code), too. This focusses only on the linear blending method, now, but you have of course nicely extended the method with the saliency-based blending approach.